docs/root/api/api.rst

```diff
@@ -9,3 +9,4 @@ API
 
   ../api-v2/api
   ../api-docs/xds_protocol
+  factory_categories
```

Comment:
I don't think this is part of API, can you move this to https://311136-65214191-gh.circle-artifacts.com/0/source/generated/docs/configuration/configuration.html?

Reply:
done